logging.go

```diff
@@ -25,9 +26,14 @@ func prefixWithReplicationId(lr LoggingReplication, raw string) string {
 	replicationId := lr.Parameters.ReplicationId
 
 	if replicationId != "" {
-		return strings.Join([]string{"[",replicationId ,"] ", raw}, "")
+		return strings.Join([]string{"[", replicationId, "] ", raw}, "")
```

Comment:
We need to come up with a stricter punishment for merges that don't run baseline go fmt - really want to lose this noise.